applications: Added guide about extending Matter bridge app (#13452)
nrfconnect/sdk-nrf: applications/matter_bridge/doc/extending_bridge.rst

.. _matter_bridge_extending:

Extending Matter Bridge application
###################################

> The Matter Bridge application presents an example implementation that supports a finite subset of bridged device types.

Reviewer: Could use similar wording as is used in the introduction of adding new Matter device types:
```suggestion
The Matter Bridge application presents an example implementation that supports only a few bridged device types.
```
